templates/login.html

```diff
@@ -105,7 +105,11 @@
     <div>
       <h4>Where can I learn more?</h4>
 
-      The <a href="https://2i2c.org/pilot">2i2c Pilot Hubs Documentation</a> has all of the information that you need to use the hub, as well as information about the deployment technology.
+      {% if custom.learn_more is defined %}
```

Comment:
We unconditionally provide a title, but if `learn_more` is either undefined / an empty string it leads to a title with no content.

I suggest:

```suggestion
      {% if "learn_more" in custom and custom["learn_more"]
```

Reply:
When left undefined, the 'else' branch was taken as I expected, and the default message was shown:

<img width="792" alt="image" src="https://github.com/2i2c-org/default-hub-homepage/assets/30430/fad3c555-1f0e-46b9-8472-8045235899d2">

This is currently deployed on https://staging.ucmerced.2i2c.cloud.

I think the content being empty when it is explicitly set to an empty string is appropriate behavior.

Reply:
Hmmm, then we would get:

```
# Where can I learn more?

<nothing here>

# What is a Jupyter Notebook?

...
```

instead of:

```
# What is a Jupyter Notebook?

...
```

`learn_more` have four possible kinds of values: undefined, null, empty string, or a non-empty string. I like to see us map these to these kinds of template renderings:

1. a header + default body text
2. a header + new body text
3. clean removal of header and default body text

To me it doesn't make sense to render the case of "a header + no body text", but I agree that there is a point to be able to render nothing (3 above).

Reply:
Opened #30 to reflect concrete suggestion that allows to render nothing - also not the learn more header - if specifying a falsy value.

Reply:
Thanks for opening #30, @consideRatio. I want to keep the time I spend on this under 30 minutes total, so with that in mind, I'm just going to merge this PR as is. While it is indeed cleaner to also handle the edge case where we explicitly set the string to "", reviewing #30 would require me to learn how jinja2 handles 'truthiness' and how that is different from 'is defined'. I am trying generally to treat review work as something serious that needs dedicated time, *and* to be really strict about how I spend my own time, to avoid side quests. To that end, I'm going to merge this one as is (based on 'If you have considered this and think whats there now makes most sense, go for it.' from https://github.com/2i2c-org/default-hub-homepage/pull/29#discussion_r1577740444), so I can close out the support ticket in a time bound manner. 

I am not opposed to #30, but don't have the time to review it myself. I hope this is an understandable choice for me to make.

I'm also reasonably convinced that with appropriate guidelines, it's totally ok to test and self-merge #30 if you want to (and think it is important enough to spend time on). I've been drafting https://github.com/2i2c-org/infrastructure/issues/3997 for a while, and finally put it up.